ulog/log_test.go

```diff
@@ -161,3 +162,18 @@ func TestSentryHookDoesNotMutatePrevious(t *testing.T) {
 	assert.NotNil(t, l2.sh.Fields())
 	assert.Equal(t, map[string]interface{}{"key": "value"}, l2.sh.Fields())
 }
+
+func TestStackTraceLogger(t *testing.T) {
+	t.Parallel()
+	testutils.WithInMemoryLogger(t, nil, func(zapLogger zap.Logger, buf *testutils.TestBuffer) {
+		log := Builder().SetLogger(zapLogger).Build()
+		err1 := errors.New("for sure")
+		err2 := errors.Wrap(err1, "it's a trap")
+		log.Error("error message", "error", err2)
+		trace := `{"level":"error","msg":"error message","stacktrace":{"log_test.go":"line:171","in_memory_log.go":"line:60","log_test.go":"line:178"`
```

Comment:
Understand the readability concerns but function name is very useful. It helps track down the root cause much faster and the information is right there whereas with the line number you almost always have to look up the file and go to the line and trace back up to see what func it is. The func names  also give a quick sense of the trace of function calls it took to get to this error.